content/docs/setup/kubernetes/upgrade/steps/index.md

```diff
@@ -146,149 +146,57 @@ $ kubectl apply -f <(istioctl kube-inject \
+### Migrating from `RbacConfig` to `ClusterRbacConfig`
 
-If you use service annotations to override global mutual TLS enablement for a service, you need to replace it with
-[authentication policy](/docs/concepts/security/#authentication-policies) and [destination rules](/docs/concepts/traffic-management/routing/destination-rules/).
+You can ignore the following if you have already migrated to `ClusterRbacConfig`. We strongly recommend
+you to migrate to `ClusterRbacConfig` if you are still using `RbacConfig`. We will stop supporting the
+`RbacConfig` in Istio 1.3.
 
-For example, if you install Istio with mutual TLS enabled, and disable it for service `foo` using a service annotation like below:
+The `RbacConfig` is deprecated in Istio 1.1 due to a [bug](https://github.com/istio/istio/issues/8825).
+The bug reduces the scope of the object to be namespace-scoped in some cases. The `ClusterRbacConfig`
+follows the exact same specification as the `RbacConfig` but with the correct cluster scope implementation.
 
-{{< text yaml >}}
-kind: Service
-metadata:
-  name: foo
-  namespace: bar
-  annotations:
-    auth.istio.io/8000: NONE
-{{< /text >}}
+To automate the migration, we developed the `convert_RbacConfig_to_ClusterRbacConfig.sh` script.
+The script is included in the [Istio installation package](/docs/setup/kubernetes/#downloading-the-release).
 
-You need to replace this with this authentication policy and destination rule (deleting the old annotation is optional)
+Download and run the script with the following command:
 
-{{< text yaml >}}
-apiVersion: "authentication.istio.io/v1alpha1"
-kind: "Policy"
-metadata:
-  name: "disable-mTLS-foo"
-  namespace: bar
-spec:
-  targets:
-  - name: foo
-    ports:
-    - number: 8000
-  peers:
----
-apiVersion: "networking.istio.io/v1alpha3"
-kind: "DestinationRule"
-metadata:
-  name: "disable-mTLS-foo"
-  namespace: "bar"
-spec:
-  host: "foo"
-  trafficPolicy:
-    tls:
-      mode: ISTIO_MUTUAL
-    portLevelSettings:
-    - port:
-        number: 8000
-      tls:
-        mode: DISABLE
+{{< text bash >}}
+$ curl -L {{< github_file >}}/tools/convert_RbacConfig_to_ClusterRbacConfig.sh | sh -
 {{< /text >}}
 
-If you already have destination rules for `foo`, you must edit that rule instead of creating a new one.
-When create a new destination rule, make sure to include other settings, i.e `load balancer`, `connection pool` and `outlier detection` if necessary.
-Finally, If `foo` doesn't have sidecar, you can skip authentication policy, but still need to add destination rule.
+The script automates the following operations:
 
-If 8000 is the only port that service `foo` provides (or you want to disable mutual TLS for all ports), the policies can be simplified as:
+1. The script creates the cluster RBAC configuration with same specification as the existing RBAC configuration
+because Kubernetes doesn't allow the value of `kind:` in a custom resource to change after it's created.
```

Comment:
Please indent multiline list items properly.
```suggestion
    because Kubernetes doesn't allow the value of `kind:` in a custom resource to change after it's created.
```